node/baremetal/README.md

```diff
@@ -0,0 +1,418 @@
+        --execution-endpoint=http://localhost:8551 \
+        --jwt-secret=PATHTO/ethereum/prysm/jwt.hex \
+        --checkpoint-sync-url=https://sync-mainnet.beaconcha.in \
+        --genesis-beacon-api-url=https://sync-mainnet.beaconcha.in \
+        --datadir=PATHTO/ethereum/prysm/data
+WorkingDirectory=/opt/ethereum/prysm
+Restart=always
+RestartSec=60
+TimeoutSec=180
+KillMode=process
+Environment="USE_PRYSM_MODERN=true"
+
+[Install]
+WantedBy=multi-user.target
+```
+
+8. Enable the new services to run with the following commands:
+
+```bash
+sudo systemctl daemon-reload
+sudo systemctl enable geth.service
+sudo systemctl enable prysm.service
+sudo systemctl start geth.service
+```
+
+The first command refreshes systemd to recognize the new services, and the second
+command enables the services to run on startup. The third command starts the
+services. You only need to call start on geth.service as it is set with
+`Wants=prysm.service` and will trigger the startup of that service.
+
+Before moving forward you can confirm the two services have started up using the
+following command for geth:
+
+```bash
+sudo systemctl status geth
+```
+
+and the following command for prysm:
+
+```bash
+sudo systemctl status prysm
+```
+
+8. Wait for the L1 node to sync. This can take a day or so to fully catch up
+   with the latest blocks.
+
+## Set up Base Node
+
+Base nodes are run as a combination of two clients. One is [OP-Geth](https://github.com/ethereum-optimism/op-geth),
+which is a forked version of Geth that does the block creation for the L2. The
+other is the [OP-Node](https://github.com/ethereum-optimism/optimism/tree/develop/op-node),
+which handles the rollup processing from L1 sources as well as the L2 -> L1 data feed.
+
+In order to run all of these as root, you will need to add `go` to the root path:
+
+1. Edit the sudo config: `sudo visudo`
+2. Locate the line staring with `Defaults    secure_path =`
+3. Add `:/usr/local/go/bin` to the end of the line
+4. Save with `^O` and exit with `^X`
+
+This will let you build the necessary packages with golang as a root user.
+
+After setting Go up to be runnable as root, Base can be set up with the correct
+directory structure and packages:
+
+1. Create the root `base` folder:
+
+```bash
+sudo mkdir /opt/base
+```
+
+2. Create the root base data folder:
+
+```bash
+sudo mkdir -p /opt/basedata/geth/chaindata
+```
+
+3. Clone the [OP-Geth repo](https://github.com/ethereum-optimism/op-geth) with the
+   latest release:
+
+```bash
+cd /opt/base
+sudo git clone https://github.com/ethereum-optimism/op-geth.git --branch v1.101200.1 --single-branch op-geth-repo
+cd op-geth-repo
+sudo git switch -c branch-v1.101200.1
+```
+
+3. Build the op-geth package and return to the root of the `base` folder
+
+```bash
+sudo go run build/ci.go install -static ./cmd/geth
+cd ../
+```
+
+4. Clone the [OP-Node repo](https://github.com/ethereum-optimism/optimism/tree/develop/op-node)
```

Comment:
Shouldn't they pull our fork?

Reply:
No our fork is not what's being used on mainnet Base.